File: totosuki/ABC/C/328.py
N, Q = map(int, input().split())
S = list(input())


B = [0] * (N)

for i in range(1, N):
  if S[i-1] == S[i]:
    B[i] = B[i-1] + 1
  else:
    B[i] = B[i-1]

for _ in range(Q):
  l, r = map(int, input().split())
  print(B[r-1] - B[l-1])

File: OCH_Tao/ABC/A/328.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const X = parseInt(input[0].split(" ")[1], 10);
const S = input[1].split(" ").map(x => parseInt(x, 10));
console.log(S.filter(x => x <= X).reduce((a, b) => a + b, 0));